doc/REST-interface.md

```diff
@@ -123,11 +123,15 @@ Returns various information about the transaction mempool.
 Only supports JSON as output format.
 Refer to the `getmempoolinfo` RPC help for details.
 
-`GET /rest/mempool/contents.json`
+`GET /rest/mempool/contents.json?verbose=<true|false>&mempool_sequence=<true|false>`
```

Comment:
Actually, since `false` is default, perhaps better to set that as the first option?
```suggestion
`GET /rest/mempool/contents.json?verbose=<true|false>&mempool_sequence=<false|true>`
```